website/package.json

```diff
@@ -16,15 +16,16 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0",
     "@docusaurus/preset-classic": "^2.0.0",
-    "@docusaurus/theme-search-algolia": "^2.0.0",
     "@docusaurus/remark-plugin-npm2yarn": "^2.0.0",
+    "@docusaurus/theme-search-algolia": "^2.0.0",
     "@mdx-js/react": "^1.6.21",
     "@svgr/webpack": "^5.5.0",
     "clsx": "^1.1.1",
     "file-loader": "^6.2.0",
     "prism-react-renderer": "^1.2.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "styled-components": "^5.3.5",
```

Comment:
Just to make sure, where do you use this lib in your code?

Reply:
Thanks for noticing this. You're right, I'm not using it at all - removed it from package.json file!